extensions/notebook/src/jupyter/serverInstance.ts

```diff
@@ -207,12 +210,36 @@ export class PerNotebookServerInstance implements IServerInstance {
 	}
 
 	private async copyKernelsToSystemJupyterDirs(): Promise<void> {
+		const DarwinPostfix = '_darwin';
+		const LinuxPostfix = '_linux';
+		const Win32Postfix = '_win32';
 		let kernelsExtensionSource = path.join(this.options.install.extensionPath, 'kernels');
 		this._systemJupyterDir = path.join(this.getSystemJupyterHomeDir(), 'kernels');
 		if (!(await this.utils.exists(this._systemJupyterDir))) {
 			await this.utils.mkDir(this._systemJupyterDir, this.options.install.outputChannel);
 		}
-		await this.utils.copy(kernelsExtensionSource, this._systemJupyterDir);
+		let platform = process.platform;
+		let subFolders = await this.utils.readDir(kernelsExtensionSource);
+		if (platform === 'win32') {
```

Comment:
Often stuff like this is handled with having the config in the package.json - that seems like a much cleaner way of handling this I would think

Reply:
So there's a much better way to do this and with some code from the powershell extension we'll be setting the powershell_executable property at runtime...

So this functionality isn't needed (and if we need it in the future, I like your idea of adding to package.json). So yeah.